packages/cli/test/commands/migrate.test.ts

```diff
@@ -324,7 +324,6 @@ describe('migrate - JS to TS conversion', async () => {
 
     const gitStatus = await git.status();
     expect(gitStatus.staged).toStrictEqual([
-      '.eslintrc.js',
```

Comment:
I think the test would fail if removing this. What is the reason for this since I don't see other changes it the CLI part?

Reply:
Its actually failing with it. The default for the test doesn't include this file